remove.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+packages="backend auth domain"
+for package in $packages; do
+  cd $package
+  echo "😎 Deleting '$package'"
+  sls remove
```

Comment:
Perhaps it would be better to use `npx sls remove` here since some folks (myself included) like to install Serverless Framework locally to use different versions of the Framework in different projects. However there is a caveat: If you don't have the `serverless` NPM module installed when running `npx sls remove`, NPM will prompt you to install a module named `sls`. So to be on the safe side, I'd suggest using `npx serverless remove`. This should hopefully cover all bases.

Reply:
Also very good catch! thanks